web/style.css

```
@@ -1,608 +1,317 @@
-  line-height: 1.65;
-  -webkit-font-smoothing: antialiased;
-}
+  --green: #149e61;
+  --green-deep: #026b3f;
+  --green-soft: rgba(20, 158, 97, 0.16);
 
-::selection {
-  background: #2c3a52;
-  color: var(--bright);
-}
+  --display: "IBM Plex Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)